Project-Euler/Project-Euler/Problems.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Project_Euler
+{
+    public static class Problems
+    {
+        private static void Problem1()
```

Comment:
Nice job!

Reply:
Just kidding, this will have to be public or you can't call it

Reply:
True, Ok.